mvp-nullobject-presenter/src/main/resources/META-INF/proguard/mvp-nullobject-presenter.pro

```diff
@@ -0,0 +1 @@
+-keep interface * extends com.hannesdorfmann.mosby3.mvp.MvpView
```

Comment:
Also maybe we can add `allowshrinking`, so interface would be removed if not used anywhere.
I think interface shouldn't be inlined without `allowoptimization` flag, it must be checked.
https://www.guardsquare.com/en/products/proguard/manual/usage#keepoptionmodifiers

Reply:
Interface is inlined regardless of `allowoptimization` flag.

Reply:
`allowshrinking` causes interface to be removed. I guess, we should not include this flag for now.